images/cross/Makefile

```diff
@@ -20,6 +20,12 @@ CACHE_IMAGES = $(IMAGES)
 
 BASEIMAGE = $(BUILD_REGISTRY)/base-$(GOARCH)
 
+if [[ "$(getconf _NPROCESSORS_ONLN)" == "1" ]]; then
```

Comment:
does this bash cope actually work top-level in a makefile?

Reply:
What do you mean? With this patch I successfully built on my single CPU system.